gulp/tasks/lodash.js

```diff
@@ -17,5 +17,5 @@ module.exports = function(gulp) {
     output: 'custom-lodash.js'
   };
 
-  gulp.task('lodash', shell.task(`lodash include=${configs.include.join(',')} -p -o ${configs.output}`));
+  gulp.task('lodash', shell.task(`lodash exports=node include=${configs.include.join(',')} -p -o ${configs.output}`));
```

Comment:
what does `exports=node` do?

Reply:
It ensures that lodash cli will build custom lodash script with export functions from nodejs api and will not use AMD or CommonJS style of exporting module. Without that parameter it did produce `define` keyword (to support AMD loaders) and was throwing some errors in console when `requirejs` was imported.

more info regarding lodash build: https://lodash.com/custom-builds